.github/workflows/ci.yml

```
@@ -20,72 +20,46 @@ jobs:
           md: [ '**.md' ]
           build: ['build/**', '.github/**/*.yml', '!.github/workflows/package-*', '**/*.targets', '**/*.props']
           shared: ['src/Shared/**', 'test/Shared/**']
-          sharedtests: ['test/OpenTelemetry.Contrib.Shared.Tests/**']
+          contrib-shared-tests: ['test/OpenTelemetry.Contrib.Shared.Tests/**']
           code: ['**.cs', '**.csproj', '.editorconfig']
-          aot: ['src/OpenTelemetry.Extensions.Enrichment/**']
           aottestapp: ['test/OpenTelemetry.AotCompatibility.TestApp/**']
-          aspnet: ['*/OpenTelemetry.Instrumentation.AspNet/**', '*/OpenTelemetry.Instrumentation.OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/**', 'examples/AspNet/**', '!**/*.md']
-          aspnetcore: ['*/OpenTelemetry.Instrumentation.AspNetCore*/**', '!**/*.md']
-          aws: ['*/OpenTelemetry.*.AWS*/**', '!**/*.md']
-          azure: ['*/OpenTelemetry.ResourceDetectors.Azure*/**', '!**/*.md']
-          eventcounters: ['*/OpenTelemetry.Instrumentation.EventCounters*/**', 'examples/event-counters/**', '!**/*.md']
+          exporter-geneva: ['*/OpenTelemetry.Exporter.Geneva*/**', '!**/*.md']
+          exporter-infuxdb: ['*/OpenTelemetry.Exporter.InfluxDB*/**', '!**/*.md']
+          exporter-instana: ['*/OpenTelemetry.Exporter.Instana*/**', '!**/*.md']
+          exporter-onecollector: ['*/OpenTelemetry.Instrumentation.OneCollector*/**', '!**/*.md']
```

```suggestion
          exporter-onecollector: ['*/OpenTelemetry.Exporter.OneCollector*/**', '!**/*.md']
```

Looks like it was wrong before too woops!